namingserver/src/main/java/org/apache/seata/namingserver/manager/NamingManager.java

```diff
@@ -17,17 +17,17 @@
 package org.apache.seata.namingserver.manager;
 
 
-import jdk.internal.net.http.common.Pair;
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
为什么进行更换依赖？
Why change dependencies?

Reply:
I think `jdk.internal.net.http.common.Pair` depends on jdk version while ` org.apache.commons.lang3.tuple.Pair` is more general.

Reply:
> I think `jdk.internal.net.http.common.Pair` depends on jdk version while ` org.apache.commons.lang3.tuple.Pair` is more general.

除此之外还有什么影响吗？如果使用后者，将增加空间大小
Is there any other impact? If you use the latter, it will increase the space size

Reply:
`jdk.internal.net.http.common.Pair` is not supported by jdk8 .besides, no more impact . I would change back if necessary

Reply:
> `jdk.internal.net.http.common.Pair` is not supported by jdk8 .besides, no more impact . I would change back if necessary

ok